Resources/ServerInfo/Rules.txt

```diff
@@ -13,70 +13,67 @@
 
 [color=#7ade00]03.[/color] This is an English only server, as we can not moderate other languages speaking in anything but English will usually lead to a single warning, then ban if you do not cease.
 
-[color=#7ade00]04.[/color] Absolutely no bigotry. This covers ableism, homophobia, racism, sexism, speciesism, and hate speech as a whole. If it's to the point where it makes someone uncomfortable and they ahelp you or an admin finds an issue with it, you will answer for it.
+[color=#7ade00]04.[/color] Absolutely no discrimination nor bigotry against any player or their characters for any reason. This covers ableism, homophobia, racism, sexism, speciesism, and hate speech as a whole. If it's to the point where it makes someone uncomfortable and they ahelp you or an admin finds an issue with it, you will answer for it.
 
 [color=#7ade00]05.[/color] No sexual content, disgusting RP, or gross statements. This is not an 18+ server. This means no erotic roleplay, emotes and messages that are sexual in nature, or overtly disgusting things no one wants to hear. Includes "technically not NSFW" jokes.
 
 [color=#7ade00]06.[/color] Do not communicate in-round information about an on-going round through multiple accounts or to other people.
 - Often referred to as metacomms, this is posting screenshots of a live round, talking about in-character events in OOC, sitting in a voice chat talking about the round, etc. It does not matter how minor it is, this is zero tolerance.
-- Do not multikey either, which is to play or spectate in the same round with multiple accounts operated by you.
+- Do not multikey, which is to play or spectate in the same round with multiple accounts operated by you.
 
-[color=#7ade00]07.[/color] Intentionally using, or abusing exploits, bugs, and external programs is not allowed.
-- From the moment you learn it's a bug you're expected to report it and never intentionally use it again.
-- External programs are things such as autoclickers, macros, and such.
+[color=#7ade00]07.[/color] Intentionally using and/or abusing exploits, bugs, or external programs are not allowed.
+- The moment you learn of a bug, you are expected to report it immediately and never intentionally use it again.
+- Some examples of external programs are as follows: autoclickers, macros, client cheats, etc.
+- Any form of cheating or hacking will immediately result in a permanent ban with no chance of appeal.
```

Comment:
Okay, this should probably be changed from "No chance of an appeal" to "No chance of appeal without a voucher."